### ISSUE TITLE
Improve selected emoji textbox UI

### ISSUE THREAD

**Issue by Substancia** [aditya-singh9/kekfinder#55]
The textbox just looks odd.
I would like to discuss this in new issue.

_Originally posted by @aditya-singh9 in https://github.com/aditya-singh9/kekfinder/issues/54#issuecomment-1264621254_

Since we can't place a cursor in this textbox, this needn't be a textbox in my opinion. We can explore more designs, like say, a left aligned text "Selected emojis: " followed by the selected emojis list (the text itself appears only if there are selected emojis)? In any way, I'd like to take away the black border of the textbox for sure.

Thoughts?

**Comment by aditya-singh9** [comment]
Iguess i can go with it.

**Comment by Substancia** [comment]
https://github.com/Substancia/kekfinder/tree/feature/improve-selected-emoji-textbox-ui

@aditya-singh9 thoughts? It's minimalistic...

**Comment by aditya-singh9** [comment]
Please attach an image or a video about the end results

**Comment by Substancia** [comment]
<img width="589" alt="Screenshot 2022-10-03 at 00 42 17" src="https://user-images.githubusercontent.com/27500147/193471971-1b9cd353-f71b-434b-a651-b74aeb824d1b.png">
<img width="620" alt="Screenshot 2022-10-03 at 00 42 47" src="https://user-images.githubusercontent.com/27500147/193471976-2f00ce5f-2f9c-4fd8-9101-7f6472b0ddd5.png">

"Selected emojis" text appears only if there are emojis selected.

**Comment by aditya-singh9** [comment]
It seems cool
But what will happen if we scroll? Like will it be there or will it float like the search bar

And please do something about those copy and delete buttons
There color looks odd

**Comment by Substancia** [comment]
Currently the selected emojis section (if visible) will float like search bar, but that's also kinda looking weird because stickiness wasn't implemented with proper styling measures (mentioned [here](https://github.com/aditya-singh9/kekfinder/issues/24#issuecomment-1264684888)). Once that is done, this should look better.

As for the copy and delete buttons, I have a fair idea, will send a screenshot (in this issue a or new one, whichever you prefer).

**Comment by Substancia** [comment]
<img width="666" alt="Screenshot 2022-10-03 at 13 50 58" src="https://user-images.githubusercontent.com/27500147/193532074-a9565e9e-d910-4828-a385-da2bc20f41cb.png">

@aditya-singh9 thoughts?

**Comment by Substancia** [comment]
<img width="630" alt="Screenshot 2022-10-03 at 13 54 30" src="https://user-images.githubusercontent.com/27500147/193532680-a77a5f8d-f0f2-4418-976a-bb497fd9f128.png">

Or maybe this (different delete icon)?

**Comment by aditya-singh9** [comment]
Seems good.

**Comment by Substancia** [comment]
[PR](https://github.com/aditya-singh9/kekfinder/pull/66) raised.
